src/viam/logging.py

```diff
@@ -43,7 +75,7 @@ def getLogger(name: str) -> logging.Logger:
     return logger
 
 
-def addHandlers(logger: logging.Logger):
+def addHandlers(logger: logging.Logger, default=False):
```

Comment:
`default` is pretty ambiguous here, I'm not sure what we mean by it exactly. Can we we rename to `use_default_handler` or something like that?